supplementary/alica_tracing/src/tracing/Trace.cpp

```diff
@@ -49,7 +62,7 @@ std::string Trace::context() const
 
 void Trace::finish()
 {
-    setTag("endTime", std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count());
+    setTag("endTime", RawTraceValue(std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count()));
```

Comment:
Wouldn't this convert the RawTraceValue to an int and then back?

Reply:
There's a `RawTraceValue` overload in this class, it should be picked.